kratos/utilities/read_materials_utility.cpp

```diff
@@ -71,6 +71,13 @@ ReadMaterialsUtility::ReadMaterialsUtility(
 /***********************************************************************************/
 /***********************************************************************************/
 
+ReadMaterialsUtility::~ReadMaterialsUtility()
```

Comment:
why do you define this in the cpp?

Reply:
Because the class had a cpp already, so I thought I could use it. I don't expect it to make a large difference in practice.